src/field_theory/is_alg_closed/classification.lean

```diff
@@ -204,6 +204,8 @@ begin
   apply classical.choice,
   rcases char_p.char_is_prime_or_zero K p with hp | hp,
   { haveI : fact p.prime := ⟨hp⟩,
+    letI : algebra (zmod p) K := zmod.algebra _ _,
```

Comment:
Note that here we don't assume `p` to be prime in the statement, so we cannot replace `[char_p K p]` with `[algebra (zmod p) K]`